flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/filemerging/FileMergingSnapshotManagerTest.java

```diff
@@ -18,19 +18,28 @@
 package org.apache.flink.runtime.checkpoint.filemerging;
 
 import org.apache.flink.api.common.TaskInfo;
+import org.apache.flink.core.fs.CloseableRegistry;
+import org.apache.flink.core.fs.FSDataInputStream;
 import org.apache.flink.core.fs.FileSystem;
 import org.apache.flink.core.fs.Path;
 import org.apache.flink.core.fs.local.LocalFileSystem;
 import org.apache.flink.runtime.checkpoint.filemerging.FileMergingSnapshotManager.SubtaskKey;
 import org.apache.flink.runtime.jobgraph.OperatorID;
 import org.apache.flink.runtime.state.CheckpointedStateScope;
 import org.apache.flink.runtime.state.filesystem.AbstractFsCheckpointStorageAccess;
+import org.apache.flink.runtime.state.filesystem.FileMergingCheckpointStateOutputStream;
 
+import org.junit.jupiter.api.Assertions;
```

Comment:
It‘s better to use AssertJ instead of Assertions.

https://flink.apache.org/how-to-contribute/code-style-and-quality-common/#:~:text=Testing-,%23,-Tooling

Reply:
Thanks for the reminder. I have replaced Assertions with AssertJ.